test/odbc/odbc_handler.h

```diff
@@ -108,30 +91,25 @@ class OdbcHandler {
 
     void BindColumns(vector<tuple<int, int, SQLPOINTER, int>> columns);
 
+    map<ServerType, ConnectionStringObject> getOdbcDrivers();
+
   private:
     // ODBC-defined SQL variables
     SQLHENV henv_{};
     SQLHDBC hdbc_{};
     SQLHSTMT hstmt_{};
     RETCODE retcode_{};
-    
-    // DB Information
-    string db_driver_{};
-    string db_server_{};
-    string db_port_{};
-    string db_uid_{};
-    string db_pwd_{};
-    string db_dbname_{};
 
-    // Build the connection string for SQLDriverConnect
-    string connection_string_{};
```

Comment:
These could be restored if OdbcHandler were only concerned with one ConnectionStringObject

Reply:
Done, restored. OdbcHandler now only concerns itself with one ConnectionStringObject.